Src/BootCamp.Chapter/Program.cs

```diff
@@ -5,8 +5,41 @@ namespace BootCamp.Chapter
     class Program
     {
         static void Main(string[] args)
-        {
-            Console.WriteLine("Hello World!");
+        {   /*
+                This is the code for Homework 1 and 2 
+                This section I have written the variables for Tom Jefferson
+
+            */
+            Console.WriteLine("This is for Homework 1");
+            int tomAge = 19;
+            float tomWeight = 50.0f;
+            string tomName = "Tom Jefferson";
+            float tomHeight = 156.5f;
+            const int cmToM = 10000;
```

Comment:
I think the purpose of this code section was to practice variables, const, etc. Not a bad thing, but expectation was to read all from console. You did that in second section, so all good :)

Reply:
Ohhh I see. I thought the the way how the homework was written was we need to enter the first one by ourselves and the second one through the console. I am sorry English isn't my first language.